common/app/model/meta.scala

```diff
@@ -37,6 +37,7 @@ trait MetaData extends Tags {
   lazy val contentType = ""
   lazy val hideUi = false
   lazy val isImmersive = false
+  lazy val isJspm = section == "crosswords"
```

Comment:
As discussed we are going to move this logic over to https://github.com/guardian/frontend/blob/master/common/app/assets/assets.scala#L129